TestFoundation/main.swift

```diff
@@ -33,4 +33,5 @@ XCTMain([
     TestNSFileManger(),
     TestNSRange(),
     TestNSXMLParser(),
+    TestNSData(),
```

Comment:
build.py needs to have TestNSData.swift added to it to make sure it builds on linux too.